### ISSUE TITLE
Adding Select via XML

### ISSUE THREAD

**Issue by rc201612** [EdonGashi/WpfMaterialForms#33]
Hi, 

Is there currently a way to add a Select field to a form from XML? If not I would be keen to maybe get started on adding something that would make this possible.

Thanks,

**Comment by mvmcjr** [comment]
@rc201612 Hey, we actually just implemented a new livereload feature on the new repo, which can support live C# file parsing. As for the XML I could have a look, but I absolutly love JSON and hate XML 😄

**Comment by rc201612** [comment]
@redbaty Hi, is this new repo something that is public? 

I am looking to get XML form definitions from ASP.NET onto the desktop and this project is brilliant and looks like it could be the solution. Supporting both XML and JSON would probably be the best way to go.

I'm prepared to assist where / if possible.

**Comment by mvmcjr** [comment]
@rc201612 Yup it is, you can [check it out here](https://github.com/WPF-Forge/Forge.Forms/tree/develop), but it still in development so it could be quite unstable. As for the JSON stuff if it fits for you then I'll take a look at it, it shouldnt be too difficult

**Comment by edongashi** [comment]
I'm thinking of separating the XML parser into a new library.

If you are interested in contributing this feature this code is where you should look into adding the parsing logic for a select element:
https://github.com/WPF-Forge/Forge.Forms/blob/develop/src/Forge.Forms/FormBuilding/FormBuilder.cs#L287

Do you have slack? We are using that to discuss about the project. Would you be so kind to invite him over @redbaty?

**Comment by mvmcjr** [comment]
Here's an invite link: https://join.slack.com/t/wpfforge/shared_invite/enQtMzAyMTg0NzYxMDkxLTk4MGEzZDBiYjQ0ZjdjYmQ0ZTBkOWVlNThhOTA0MWFmZDczY2U3MzI1N2Q1YzQyMjg1YjdiYmJiYTA4ZmRmNTg
